plugins/MauticTagManagerBundle/Controller/TagController.php

```diff
@@ -485,6 +485,232 @@ public function viewAction(Request $request, TagDependencies $tagDependencies, $
         ]);
     }
 
+    /**
+     * Merge two tags together.
+     */
+    public function mergeAction(Request $request, int $objectId): Response
```

Comment:
```suggestion
    public function mergeAction(Request $request, int $objectId, TagModel $model): Response
```
As we have autowiring, we should use DI.